query-string/query-string.d.ts

```diff
@@ -9,14 +9,14 @@ declare module "query-string" {
      * Leading ? or # are ignored, so you can pass location.search or location.hash directly.
      * @param str
      */
-    export function parse(str: string): any;
+    export function parse(str: string): {[key: string]: string};
```

Comment:
According to `{likes: ['cake', 'icecream'], name: 'bob'}` of document demo, it should be `function parse(str: string): {[key: string]: string | string[]};`